### PR TITLE
Update Chartbeat vendor config

### PR DESCRIPTION
**Pull request by breauxc**
This pull request addresses issue #1985, and is dependent on the implementation of the totalEngagedTime variable added in PR #1818 for issue #1296.

### CODE REVIEW

**Comment by rudygalfi** [comment]
I noticed the changes also include adding a bunch of stuff that's not related to the engaged time metrics. Does it make sense to fork those independent changes into a separate PR so they get resolved faster?

File: extensions/amp-analytics/0.1/vendors.js [hunk 1]
```diff
@@ -89,14 +89,23 @@ export const ANALYTICS_CONFIG = {
         'g3=${sponsorName}&' +
         'g4=${contentType}&' +
         'x=${scrollTop}&' +
+        'y=${scrollHeight}&' +
         'w=${screenHeight}&' +
```
Comment (avimehta) [inline extensions/amp-analytics/0.1/vendors.js]: just curious, you are assigning screenHeight to w. Should it be h instead? also do you not care about the screen width?

File: extensions/amp-analytics/0.1/vendors.js [hunk 2]
```diff
@@ -89,14 +89,23 @@ export const ANALYTICS_CONFIG = {
...
+        'C=2&' +
+        'R=1&' +
+        'W=0&' +
+        'I=0&' +
+        'c=120',
       'baseSuffix': '&_',
-      'interval': '${host}${basePrefix}&${baseSuffix}',
-      'anchorClick': '${host}${basePrefix}&${baseSuffix}'
+      'interval': '${host}${basePrefix}${baseSuffix}',
+      'anchorClick': '${host}${basePrefix}${baseSuffix}'
```
Comment (avimehta) [inline extensions/amp-analytics/0.1/vendors.js]: please add trailing comma. It is apparently valid in javascript now and reduces diffs :).

File: extensions/amp-analytics/0.1/vendors.js (hunk 1 above)
Comment (breauxc) [inline extensions/amp-analytics/0.1/vendors.js]: Thanks for flagging this; we're actually using 'w' for windowHeight, which is consistent with screenHeight for the purposes of mobile devices, but we're going to remove it for now.

**Comment by breauxc** [comment]
Made a couple of changes to address the comments (Thanks!). We're content to leave this as a PR until the engaged time changes go through, with the hopes that this can be merged in and released concurrently.

**Comment by cramforce** [comment]
@avimehta gave LGTM

**Pull request merged**